XenModel/XenAPI/FriendlyErrorNames.resx

```diff
@@ -1958,6 +1958,9 @@ Authorized Roles: {1}</value>
   <data name="VGPU_DESTINATION_INCOMPATIBLE" xml:space="preserve">
     <value>The VGPU is not compatible with any PGPU in the destination.</value>
   </data>
+  <data name="VGPU_TYPE_NOT_COMPATIBLE" xml:space="preserve">
+    <value>You try to create a VGPU that is not compatible with existing types on the VM.</value>

```

Comment:
"You tried ..."

Reply:
I will add it to the XAPI PR I'm working on.